requirements.txt

```diff
@@ -17,6 +17,7 @@ pytest-mock==1.10.0
 pytest-cov==2.5.1
 tabulate==0.8.2
 coinmarketcap==5.0.3
+ujson==1.35
```

Comment:
This increases the requirements on windows hugely ... 
```
error: Microsoft Visual C++ 14.0 is required. Get it with "Microsoft Visual C++ Build Tools": http://landinghub.visualstudio.com/visual-cpp-build-tools ... 
```
(link does not work anymore ...) - but it requires installing visual studio.

Not sure that's worth the speedup for something rarely used during day-to-day operations...

Reply:
wow - yeh, totally not worth it. 
The failed test was on cached data, not needed for the loading speedup. 

Agree its not suitable if windows users need to install the world, how are they managing other compiled c, such as TA-Lib?

Reply:
ta-lib we have a instruction to install egg (it's not straightforward on linux either as you need the OS package ) ... 
didn't find one for this library though - but i'm at work, so didn't look too hard

Reply:
are these them?
https://www.lfd.uci.edu/~gohlke/pythonlibs/#ujson